bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/action/VeluxActions.java

```diff
@@ -91,12 +91,11 @@ public void setThingHandler(@Nullable ThingHandler handler) {
      * @throws IllegalArgumentException if actions is invalid
      * @throws IllegalStateException if anything else is wrong
      */
-    public static Boolean rebootBridge(@Nullable ThingActions actions)
-            throws IllegalArgumentException, IllegalStateException {
-        if (!(actions instanceof IVeluxActions)) {
+    public static Boolean rebootBridge(ThingActions actions) throws IllegalArgumentException, IllegalStateException {
```

Comment:
Please also restore the `@Nullable` on actions parameter.

Reply:
Ok. ..

.. but this makes me wonder if all the other arguments in these method calls should also be declared as nullable, and therefore also include respective null checks in the code?

e.g. I suppose it is technically feasible that a user might call `moveMainAndVane(null, null, null)` with null argument from within a rule; or even with too many or too few arguments ..

Reply:
Yes, not wrong.
The risk is just higher with the actions parameter as it requires a particular code added before the call. If the user makes an error, null will be sent.
But, yes, you're right, it would be more secure to also check the other parameters.

Reply:
Ok. Done.